scripts/check-merge-conflicts.sh

```diff
@@ -0,0 +1,11 @@
+result=$(find ./ -type f \( -iname \*.json -o -iname \*.sh \) | while read file_path; do
```

Comment:
Won't our other files conflict? For example dockerfile?

Reply:
Addressed.

Reply:
Thanks!